### PR TITLE
Bump to node 4.1.0

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

**Comment by charypar** [comment]
:+1:

**Comment by imranolas** [comment]
You might also want to update the babel transforms, so that we're only transforming features that aren't natively supported.

**Comment by charypar** [comment]
@imranolas good point, any good docs for that?

**Comment by ironsidevsquincy** [comment]
oh, yeah, that's cool. I'll update

**Comment by imranolas** [comment]
Here's a list of the natively supported es6 features. https://nodejs.org/en/docs/es6/
These are the babel transformers. https://babeljs.io/docs/advanced/transformers/

You should be able to use the `whitelist` option to declare *only* the transformations you want to apply. It starts to get a little manual at this point because babel doesn't yet have the option of choosing a compile target other than es5.

File: circle.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ machine:
   services:
     - docker
```
Comment (matthew-andrews) [inline circle.yml]: can also delete `docker`

File: server/app.js [hunk 2]
```diff
@@ -1,2 +1,13 @@
-require('babel/register');
+require('babel/register')({
+	whitelist: [
+		'es6.arrowFunctions',
```
Comment (ironsidevsquincy) [inline server/app.js]: annoyingly need this, or else this wonderful error is produced

screen shot 2015-09-21 at 14 30 32

**Pull request merged**